index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Trybewarts</title>
+  <link rel="stylesheet" href="styles.css">
+</head> 
+<body>
+  <!-- cria a caixa verde no topo da tela -->
+  <div class="header">
+    <img class="trybewarts-header-logo" src="./images/trybewarts-header-logo.svg" alt="">
+    <h1 id="trybewarts-header-title">Trybewarts</h1>
+    <form class="trybewarts-login">
+        <input id="input-email" type="email" name="email" placeholder="Email">
+        <input id="input-senha" type="password" name="password" placeholder="Senha">
+        <button id="botao-entrar" type="button">Entrar</button>
+    </form>
+  </div>
+  <main>
+    <form id="evaluation-form" action="">
+      <!-- requisito 8 -->
+      <input type="text" id="input-name" placeholder="Nome">
+      <input type="text" id="input-lastname" placeholder="Sobrenome">
+      <input type="email" id="input-email" placeholder="Email">
+      <!-- requisito 12 -->
+      <div class="familias">
+        <label id="label-family">Qual sua família?</label>
+        <label for="Frontend">Frontend</label>
+        <input name="family" type="radio" value="Frontend">
+        <label for="Backend">Backend</label>
+        <input name="family" type="radio" value="Backend"> 
+        <label for="FullStack">FullStack</label>
+        <input name="family" type="radio" value="FullStack"> 
+      </div>  
+      <!-- requisito 9 -->
+      <select name="" id="house">
+        <option value="" disabled selected>selecione sua casa</option>
+        <option value="Gitnória" id="gitnoria-house">Gitnória</option> 
+        <option value="Reactpuff" id="reactpuff-house">Reactpuff</option> 
+        <option value="Corvinode" id="corvinode-house">Corvinode</option> 
+        <option value="Pytherina" id="pytherina-house">Pytherina</option> 
+      </select>
+      <!-- requisito 13 -->
+      <div class ="content">
+        <label id="label-content">Qual conteúdo você está com mais vontade de aprender?</label>
+        <input type="checkbox" value="HoFs">
+        <label for="Hofs">Hofs</label>
+        <input type="checkbox" value="Jest">
+        <label for="Jest">Jest</label>
+        <input type="checkbox" value="Promises">
+        <label for="Promisses">Promisses</label>
+        <input type="checkbox" value="React">
+        <label for="React">React</label>
+        <input type="checkbox" value="SQL">
+        <label for="SQL">SQL</label>
+        <input type="checkbox" value="Python">
+        <label for="Python">Python</label>
```

Comment:
Mano só pra lembrar que o for do <label> tem que ter o mesmo conteúdo do name do <input> pra ele funcionar corretamente.
```suggestion
        <input name="hofs" type="checkbox" value="HoFs">
        <label for="hofs">Hofs</label>
        <input name="jest" type="checkbox" value="Jest">
        <label for="jest">Jest</label>
        <input name="promisses" type="checkbox" value="Promises">
        <label for="promisses">Promisses</label>
        <input name="react" type="checkbox" value="React">
        <label for="react">React</label>
        <input name="sql" type="checkbox" value="SQL">
        <label for="sql">SQL</label>
        <input name="python" type="checkbox" value="Python">
        <label for="python">Python</label>
```
Dessa forma ao clicar com o mouse tanto no radio quanto na label ativa o radio desejado.

Reply:
Ok, acabei de modificar. Obrigado pela sugestão. Só atente que o que deixa a label clicável é a propriedade "Id" e não a "name"